core-bundle/src/Resources/config/services.yml

```diff
@@ -496,6 +496,20 @@ services:
             - '%contao.prepend_locale%'
         public: true
 
+    contao.search.indexer.default:
```

Comment:
This should be `Contao\CoreBundle\Search\Indexer\DefaultIndexer` (we will be using the class names as service IDs for all new services).

Reply:
Right, changed in 1b715d395b939d6f9924d1f7a66e97f85c0de884.